src/steamship/data/tag_utils.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any, Optional
+
+from steamship import File
+
+
+def first_value(file: File, tag_kind: str, value_key: str) -> Optional[Any]:
```

Comment:
If adding this, might be nice to have an optional tag_name as well

Reply:
done. wdyt?